### DISCUSSION TITLE
How to add custom blocks to apiary.

### DISCUSSION THREAD

**Discussion by CalvinBard** [Resourceful-Bees/ResourcefulBees#96] [General]
I can't seem to find how to add blocks to the apiary structure whitelist.
I've seen you guys talk about a taglist but cant find it anywhere.

**Comment by Epic428** [comment]
https://minecraft.gamepedia.com/Data_Pack

https://minecraft.gamepedia.com/Tag

those are tutorials for how to make tags and include them in a data pack which is a normal vanilla function. All you need to do is add a `valid_apiary` item and block tag. Set replace to false and simply list whatever blocks/tags you want added.
